remove-node.yml

```diff
@@ -3,11 +3,11 @@
   become: no
   gather_facts: no
   tasks:
-    - name: "Check ansible version >=2.7.8"
+    - name: "Check ansible version >=2.9.1"
```

Comment:
This is a radical change. Is it not possible to adjust our checks (like cores checking) to be case insensitive so we don't need to force people off 2.7.x immediately?

Reply:
This is open for discussion. 2.9 is very new, so it's maybe a little early to require 2.9.

One option is to add the fixes but not modify the assert and the `requirements.txt`, the downside with that is that all CI will continue to run on 2.7.

Reply:
We had a slack discussion about this, the outcome is that we will keep 2.7 as a requirement for now, but try to make things compatible with 2.8 and 2.9 without CI.